### PR TITLE
added file camelCase.js

### PR DESCRIPTION
**Pull request by EleonoraDellamico**
I created a new file and I did a simple exercise.  In a sequence of words in CamelCase as a string of letters, you have determined the number of words in the string "saveChangesInTheEditor".

Here you can find the link from the web page **hacker rank** with more details.
[HackerRank problem](https://www.hackerrank.com/challenges/camelcase/problem).

@LuciaMezquida @ferminhg

### CODE REVIEW

File: hackerank/js-practice/camelCase.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+   let count = 1;
+   for(let i in s){
+    if(s.charAt(i) === s.charAt(i).toUpperCase()){
+      count++
+    }
+   }
+   return count
+}
+
+console.log(camelcase("saveChangesInTheEditor"));
```
Comment (ferminhg) [inline hackerank/js-practice/camelCase.js]: It's a good practice to add a new line empty at the of file. 

Thas why GH shows the ⛔️ icon in this line.

Comment (ferminhg) [inline hackerank/js-practice/camelCase.js]: Could you add more examples? and what is the expected result?

Example:
```javascript
if camelcase("WoPWop") == 3 {
  console.log(👍);
else
  console.log(👎);
```

File: hackerank/js-practice/camelCase.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function camelcase(s) {
+   let count = 1;
+   for(let i in s){
+    if(s.charAt(i) === s.charAt(i).toUpperCase()){
+      count++
```
Comment (ferminhg) [inline hackerank/js-practice/camelCase.js]: I recommend using a code style plugging to be more consistent in your code styling.
- add `;` at the of line or not
- add space before `{` 
- add space after `(` 

These seem the most used:
- [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify)
- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode )

**Review by ferminhg — COMMENTED** [review]
Good job, these are nitpicking comments for practice PR updates.

File: hackerank/js-practice/camelCase.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+function camelcase(s) {
+   let count = 1;
+   for(let i in s){
```
Comment (LuciaMezquida) [inline hackerank/js-practice/camelCase.js]: Even though it's completely understandable, it would be better to use more descriptive variables

**Comment by LuciaMezquida** [comment]
Good job, Eli. I'm proud of your improvement

File: hackerank/js-practice/camelCase.js (hunk 3 above)
Comment (EleonoraDellamico) [reply hackerank/js-practice/camelCase.js]: I agree I did a new change but I'm not sure if you could see it or not

File: hackerank/js-practice/camelCase.js (hunk 2 above)
Comment (EleonoraDellamico) [reply hackerank/js-practice/camelCase.js]: Perfect, I really don't know why but sometimes prettier and beautiful don't work properly. Let me know if you see the changes.

File: hackerank/js-practice/camelCase.js (hunk 1 above)
Comment (EleonoraDellamico) [reply hackerank/js-practice/camelCase.js]: Ok, perfect I see it only now. Thank you for the tip.

Comment (EleonoraDellamico) [reply hackerank/js-practice/camelCase.js]: the goal is to know the number of words inside a camelcase string, I'm thinking about how to structure a different example.

**Review by ferminhg — APPROVED** [review]

**Pull request merged**